Makefile

```diff
@@ -61,6 +61,15 @@ install-op: ## Build and install the op-reth binary under `~/.cargo/bin`.
 build: ## Build the reth binary into `target` directory.
 	cargo build --bin reth --features "$(FEATURES)" --profile "$(PROFILE)"
 
+.phone: build-reproducible
+build-reproducible: ## Build the reth binary into `target` directory with reproducible builds.
+	SOURCE_DATE_EPOCH=1724346102 \
```

Comment:
This is just hardcoded here for testing purposes and can be offloaded to an env variable instead. But the reason to pin it here because `reth --version` embeds the build time and date info which changes the binary measurement accordingly.
This way, it would be fixed to a specific timestamp and removes this factor from non-reproducibility

Reply:
It might be preferable to remove the build time altogether, instead of needing to remember to update it manually before building.

Reply:
I am not aware of a way to completely exclude the build date from rust binaries. That's why there is this env variable that can be set manually if you want to fix the date, otherwise it would be fetching the current build date upon each build.
Would be interested if you can share with me an alternative to remove the build date from the binary completely?

Reply:
mb let's set it to `git log -1 --pretty=%ct` or `0`?

Reply:
@rkrasiuk yes, setting it to `git log -1 --pretty=%ct` is a better approach. Thank you